components/clarity-vscode/package.json

```diff
@@ -83,6 +83,18 @@
             "order": 0,
             "description": "Allow auto completion for native and user-defined functions"
           },
+          "clarity-lsp.completionSmartParenthesisWrap": {
+            "type": "boolean",
+            "default": true,
+            "order": 0,
+            "description": "If set to true, the completion won't wrap a function in parenthesis if an opening parenthesis is already there"
+          },
+          "clarity-lsp.completionIncludeNativePlaceholders": {
+            "type": "boolean",
+            "default": true,
+            "order": 0,
+            "description": "If set to true, the completion of native functions will include arguments placeholder. If set to false, it will only do so for functions interacting with variables, maps, fts and nfts."
```

Comment:
```suggestion
            "description": "If set to true, the auto-completion of native functions will include arguments placeholder. If set to false, it will only do so for functions interacting with variables, maps, FTs, and NFTs."
```